backend/.env.example_local

```diff
@@ -12,4 +12,4 @@ AIRTABLE_API_KEY=TODO
 AIRTABLE_BASE_ID=appKRi7YYHy6CBar8
 SENDGRID_API_KEY=TODO
 SENDGRID_CONTACTS_LIST_ID=TODO
-
+LOGGER_LEVEL=debug
```

Comment:
Just added that for socket.io debugging. So we we don't have a lot of unnecessary logs in PROD. But since this is just .env.example anyone can change it to what they want.